huaweicloud/services/swr/resource_huaweicloud_swr_repository.go

```diff
@@ -60,7 +59,8 @@ func ResourceSWRRepository() *schema.Resource {
 					),
 					validation.StringDoesNotMatch(
 						regexp.MustCompile(`_{3,}?|\.{2,}?|-{2,}?`),
-						"Periods, underscores, and hyphens cannot be placed next to each other. A maximum of two consecutive underscores are allowed.",
+						"Periods, underscores, and hyphens cannot be placed next to each other."+
+							"A maximum of two consecutive underscores are allowed.",
```

Comment:
it's better to add a space

```suggestion
						"Periods, underscores, and hyphens cannot be placed next to each other. "+
							"A maximum of two consecutive underscores are allowed.",
```